buildtest/utils/file.py

```diff
@@ -11,60 +11,62 @@
 import os
 import logging
 
-from buildtest.exceptions import BuildTestError
+logger = logging.getLogger(__name__)
 
 
 def is_file(fname):
     """This method will check if file exist and if not found throws an exception.
 
     :param file: file path
-    :type file: string, required
+    :type file: str, required
 
-    :raises BuildTestError: sub-class of Exception
     :return: returns True if file exists otherwise terminates with an exception
-    :rtype: True if successful, otherwise throws an exception
+    :rtype: bool
```

Comment:
Should this be rtype (does it mean return type or is it a typo?)

Reply:
Yea this is return type, there are built-in types atleast documented in https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html i dont think it's an official standard from docstring but what happens is this formatting helps document all variables and return type that makes API docs look much better. 

Some IDEs have built-in support to auto-fill some sections atleast PyCharm can do this. https://www.jetbrains.com/help/pycharm/using-docstrings-to-specify-types.html